doc_src/cmds/function.rst

```diff
@@ -56,7 +56,7 @@ By using one of the event handler switches, a function can be made to run automa
 
 - ``fish_posterror``, which is emitted right after executing a command with syntax errors. The commandline is passed as the first parameter.
 
-- ``fish_postexec``, which is emitted right after executing an interactive command. The commandline is passed as the first parameter. Not emitted if command is empty.
+- ``fish_postexec``, which is emitted right after executing an interactive command. The commandline is passed as the first parameter. The second parameter indicates whether the value of `$status` is a holdover (for example, backgrounded jobs and variable assignments do not affect `$status`). Not emitted if command is empty.
```

Comment:
Also here two double backticks around `$status`. We should have a linter for this - Sphinx happily renders `<cite>$status</cite>`.

Reply:
Done